src/cascadia/WpfTerminalControl/TerminalContainer.cs

```diff
@@ -113,10 +113,23 @@ private get
                 {
                     this.connection.TerminalOutput -= this.Connection_TerminalOutput;
                 }
-
+                this.Connection_TerminalOutput(this, new TerminalOutputEventArgs("\u001b[2J")); //clear screen

```

Comment:
C# supports `\x1b` for strings btw, so there's no need for the rather verbose `\u001b`.